code/modules/mob/living/carbon/human/npcs.dm

```diff
@@ -5,9 +5,8 @@
 	item_color = "punpun"
 	species_restricted = list("Monkey")
 
-/mob/living/carbon/human/monkey/punpun/New()
-	..()
-	spawn(1)
-		name = "Pun Pun"
-		real_name = name
-		equip_to_slot(new /obj/item/clothing/under/punpun(src), slot_w_uniform)
\ No newline at end of file
+/mob/living/carbon/human/monkey/punpun/Initialize(mapload)
+	..(mapload, "Monkey")
```

Comment:
I'm not sure passing any arguments here is necessary.

Reply:
Yup, pun pun worked fine without em!